src/compile/build.ts

```diff
@@ -426,7 +426,7 @@ async function afterSuccessfulBuilt(lastStep: Step, skipped: boolean) {
     const configuration = vscode.workspace.getConfiguration('latex-workshop', vscode.Uri.file(lastStep.rootFile))
     // If the PDF viewer is internal, we call SyncTeX in src/components/viewer.ts.
     if (configuration.get('view.pdf.viewer') === 'external' && configuration.get('synctex.afterBuild.enabled')) {
-        const pdfFile = lw.file.getPdfPath(lastStep.rootFile)
+        const pdfFile = vscode.Uri.parse(lw.file.getPdfPath(lastStep.rootFile))
```

Comment:
Same as below, rename and parse.

Reply:
This has nothing to do with .synctex.gz and should be fixed at the opportunity to modify the pdf.

Reply:
I don't get what you mean.